lib/features/search/presentation/ui/widgets/search_bottom_drawer.dart

```diff
@@ -0,0 +1,234 @@
+            .add(SearchBottomDrawerSwipedDown());
+      },
+      child: Stack(
+        children: <Widget>[
+          BlocBuilder<SearchBottomDrawerBloc, SearchBottomDrawerState>(
+            builder: (context, state) {
+              if (state is SearchBottomDrawerOpened) {
+                return AnimatedPositioned(
+                    curve: Curves.easeInOut,
+                    duration: Duration(milliseconds: 200),
+                    left: 0,
+                    bottom: -95,
+                    child: DrawerContents());
+              }
+              return AnimatedPositioned(
+                  curve: Curves.easeInOut,
+                  duration: Duration(milliseconds: 200),
+                  left: 0,
+                  bottom: -232,
+                  child: DrawerContents());
+            },
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class DrawerContents extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    double height = MediaQuery.of(context).size.height;
+    double width = MediaQuery.of(context).size.width;
+
+    return ClipRRect(
+      borderRadius: BorderRadius.all(Radius.circular(10.0)),
+      child: Container(
+        color: Colors.redAccent,
+        width: width,
+        height: height / 3 + 60,
+        child: Padding(
+          padding: const EdgeInsets.symmetric(vertical: 2.0),
+          child: Column(
+            children: <Widget>[
+              Icon(
+                Icons.keyboard_arrow_up,
+                size: 20,
+              ),
+              Text(
+                "Place Details",
+                style: TextStyle(
+                    fontSize: 17,
+                    fontWeight: FontWeight.bold,
+                    color: Colors.white),
+              ),
+              PlaceDetails(),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class PlaceDetails extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<DrawerContentsBloc, DrawerContentsState>(
+      builder: (BuildContext context, DrawerContentsState state) {
+        if (state is DrawerContentsLoaded) {
+          return PlaceCard(
+              placePhoto: state.photo, placeEntity: state.placeEntity);
+        } else {
+          return Padding(
+              padding: const EdgeInsets.all(10.0),
+              child: Text(
+                "No places searched yet...",
+                style: TextStyle(
+                  fontSize: 20,
+                  fontWeight: FontWeight.bold,
+                ),
+              ));
+        }
+      },
+    );
+  }
+}
+
+class PlaceCard extends StatelessWidget {
+  final File placePhoto;
+  final PlaceEntity placeEntity;
+
+  const PlaceCard({Key key, this.placePhoto, this.placeEntity})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final navScreenBloc = BlocProvider.of<NavigationScreenBloc>(context);
```

Comment:
Not used. Delete ??  Also, I was wondering if this duplication can be minimized  but lets think about that later.

Reply:
Deleted and also the commented out code along with it